pkg/backends/cloudwatch/cloudwatch.go

```diff
@@ -57,25 +60,23 @@ func NewClient(namespace, transport string, disabled gostatsd.TimerSubtypes, log
 	if err != nil {
 		return nil, err
 	}
-	sess, err := session.NewSession(&aws.Config{
-		HTTPClient: httpClient.Client,
-	})
+	cfg, err := config.LoadDefaultConfig(context.TODO(), config.WithHTTPClient(httpClient.Client))
```

Comment:
Rather than `context.TODO` which we typically use when we're unclear what context to use and intend to update in the future, we can also do `context.Background` which is typically for empty root contexts with no values or cancellation

Reply:
Thanks, I was going off what the migration guide had but that is better